tests/unit/multiclass/BaggingMachine_unittest.cc

```diff
@@ -1,37 +1,78 @@
-#include "machine/MockMachine.h"
+#include "../utils/Utils.h"
```

Comment:
please dont do relative includes :(
i know that there are places where it is done the same way but 
```
#include "utils/Utils.h"
```
should do it.